switch_utils/switch_utils.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __KRNLMON_UTILS_H__
+#define __KRNLMON_UTILS_H__
+
+#include <assert.h>
+#include "zlog/zlog.h"
+#include "zlog/category.h"
+
+#define krnlmon_assert                  assert
+#define krnlmon_snprintf                snprintf
+
+#define DEFAULT_ZLOG_CFG_FILE           "switch_utils/krnlmon-zlog-cfg"
```

Comment:
Where is the configuration file stored on the target system, and what determines its location?
Is the location fixed, or is it something the customer can change at run time?

Reply:
For now, it's available under switch_utils. But you bring a good point that we should keep this in target system if user removed the source code, maybe under install directory or something similar. I have not tried changing it at run time. I will play around this and add this functionality if supported by zlog in next PR